### PR TITLE
Better rescue for runtime exceptions

### PR DESCRIPTION
**Pull request by scharrier**
With this PR, following suggestion of Paulr on #31 we add an error handler to display a more user friendly message when something bad happens.

It also adds an issue template so we guide our users through the process of informing us of a runtime error they had.

### CODE REVIEW

File: lib/bump/cli/commands/base.rb [hunk 1]
```diff
@@ -89,6 +89,13 @@ def with_errors_rescued
           yield
         rescue HTTP::Error, Errno::ENOENT, SocketError => error
           abort "Error: #{error.message}"
+        rescue => error
+          warn "An error has occurred. We don't monitor what happens on your computer, so we haven't been notified."
```
Comment (scharrier) [inline lib/bump/cli/commands/base.rb:93]: I'm not sure of this second sentence, but I haven't had a better idea.

File: .github/ISSUE_TEMPLATE/cli-runtime-error.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: "[ERROR MESSAGE]"
+labels: ""
+assignees: ""
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Command**
+Paste here the exact command (with args and options) you typed that generated the bug.
```
Comment (paulRbr) [inline .github/ISSUE_TEMPLATE/cli-runtime-error.md:13]:
```suggestion
Paste here the exact command executed (with arguments and options) which led to the bug.
```

File: lib/bump/cli/commands/base.rb (hunk 1 above)
Comment (paulRbr) [inline lib/bump/cli/commands/base.rb:93]:
```suggestion
          warn "An unexpected error occurred. Sorry about that!"
          warn "We don't monitor errors raised by the CLI running on your computer, so we have not been notified."
```

File: .github/ISSUE_TEMPLATE/cli-runtime-error.md (hunk 2 above)
Comment (scharrier) [reply .github/ISSUE_TEMPLATE/cli-runtime-error.md]: :+1:

File: lib/bump/cli/commands/base.rb (hunk 1 above)
Comment (scharrier) [reply lib/bump/cli/commands/base.rb]: :+1:

**Review by paulrbr-fl — APPROVED** [review]

**Review by paulRbr — APPROVED** [review]
Looks good, thanks!

**Review by Polo2 — APPROVED** [review]
LGTM !

I hope we'll receive some contributions to understand issues in several development environments, even if I hope runtime exceptions does not happen too frequently !

**Pull request merged**